app/main.py

```diff
@@ -1,6 +1,30 @@
-# write your imports here
+from app.people.cinema_staff import Cleaner
+from app.people.customer import Customer
+from app.cinema.bar import CinemaBar
+from app.cinema.hall import CinemaHall
 
 
-def cinema_visit(customers: list, hall_number: int, cleaner: str, movie: str):
-    # write you code here
-    pass
+def cinema_visit(
+        customers: list,
+        hall_number: int,
+        cleaner: str,
+        movie: str
+) -> None:
+    bar_as_obj = CinemaBar()
```

Comment:
```suggestion
    bar_as_obj = CinemaBar()
```
Can you explain why you need _as_obj suffix in this case? I do not see any variable names bar
The same questions to hall_as_obj

Reply:
I named all variable in one style. If it is problem, i will renamed bar and hall.